cli/exchange/org.go

```diff
@@ -195,15 +195,64 @@ func OrgDel(org, userPwCreds, theOrg string, force bool) {
 		cliutils.ConfirmRemove(msgPrinter.Sprintf("Warning: this will also delete all Exchange resources owned by this org (nodes, services, patterns, etc). Are you sure you want to remove user %v from the Horizon Exchange?", theOrg))
 	}
 
-	// Search exchange for org and delete it, throw error if not found.
-	httpCode := cliutils.ExchangeDelete("Exchange", cliutils.GetExchangeUrl(), "orgs/"+theOrg, cliutils.OrgAndCreds(org, userPwCreds), []int{204, 404})
+	// Search exchange for org, throw error if not found.
+	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/"+theOrg, cliutils.OrgAndCreds(org, userPwCreds), []int{200, 404}, nil)
 	if httpCode == 404 {
-		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("org %s not found.", theOrg))
-	} else {
-		msgPrinter.Printf("Organization %v is successfully removed.", theOrg)
-		msgPrinter.Println()
+		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("org '%s' not found.", theOrg))
+	}
+
+	// check if the agbot specified by -a exist or not
+	CheckAgbot(org, userPwCreds, agbot)
+
+	if agbot == "" {
+		// if -a is not specified, it go get the first agbot it can find
+		ag := GetDefaultAgbot(org, userPwCreds)
+		if ag == "" {
+			msgPrinter.Printf("No agbot found in the Exchange.")
+			msgPrinter.Println()
+			return
```

Comment:
It's okay there is no agbot found. Should not exit out. The code should skip the pattern and policy deletion part for agbot and proceed to delete the org.